doc/api/util.md

```diff
@@ -1249,6 +1249,23 @@ util.types.isMapIterator(map.entries());  // Returns true
 util.types.isMapIterator(map[Symbol.iterator]());  // Returns true
 ```
 
+### util.types.isModuleNamespaceObject(value)
+<!-- YAML
+added: v10.0.0
+-->
+
+* Returns: {boolean}
+
+Returns `true` if the value is an instance of a [Module Namespace Object][].
+
+For example:
```

Comment:
Drop this line.

Reply:
as much as i hate that line, every other one has that line.

Reply:
I can open another or after this to remove it from all of the docs at once.